cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -1871,14 +1870,15 @@ impl_runtime_apis! {
 			}
 
 			use cumulus_pallet_session_benchmarking::Pallet as SessionBench;
+			use xcm_config::{MaxAssetsIntoHolding, WestendLocation, AssetHubWestend, AssetHubId};
 			impl cumulus_pallet_session_benchmarking::Config for Runtime {}
 
 			parameter_types! {
 				pub ExistentialDepositAsset: Option<Asset> = Some((
-					WestendLocation::get(),
+					AssetHubWestend::get(),
 					ExistentialDeposit::get()
 				).into());
-				pub const RandomParaId: ParaId = ParaId::new(43211234);
+				pub const SiblingAssetHubId: ParaId = ParaId::new(AssetHubId::get());
```

Comment:
AssetHub does not know any other sibling AssetHub, `RandomParaId` is good here
```suggestion
				pub const RandomParaId: ParaId = ParaId::new(43211234);
				pub RandomParaLocation: Location = ParentThen(Parachain(RandomParaId::get().into()).into()).into();
```